packages/constants/src/constants.ts

```diff
@@ -79,3 +79,8 @@ export const METADATA_MAP: Record<TORUS_LEGACY_NETWORK_TYPE, string> = {
 
 export const FND_SERVER = "https://fnd.web3auth.io";
 export const SESSION_SERVER = "https://session.web3auth.io";
+
+export const KEY_TYPE = {
+  SECP256K1: "secp256k1",
+  ED25519: "ed25519",
+} as const;
```

Comment:
```suggestion
export enum KEY_TYPE {
  SECP256K1 = "secp256k1",
  ED25519 = "ed25519",
}
```
Why not like this? And then you can do `export type WEB3AUTH_KEY_TYPE = KEY_TYPE;` (instead of `export type WEB3AUTH_KEY_TYPE = (typeof KEY_TYPE)[keyof typeof KEY_TYPE];`).

Reply:
Or simply `export type KEY_TYPE = "secp256k1" | "ed25519"`?